packages/sdk-components-react/src/vimeo.tsx

```diff
@@ -248,10 +257,19 @@ const useVimeo = ({
     ) {
       return;
     }
+
     if (showPreview) {
-      loadPreviewImage(elementRef.current, options.url).then(
-        setPreviewImageUrl
-      );
+      if (loadPreviewOnceRef.current) {
+        return;
+      }
+
+      loadPreviewOnceRef.current = true;
+
+      loadPreviewImage(elementRef.current, options.url)
+        .then(setPreviewImageUrl)
+        .catch(() => {
+          /* DO NOTHING */
```

Comment:
not even a console.error? :)

Reply:
fetch errors are seen in console without log